lib/cmd/common/build.js

```diff
@@ -95,6 +97,10 @@ function validateArgs(args) {
       args.version = '6.0';
     }
   }
+
+  if (!args.environment) {
```

Comment:
I believe this is only required if the target of fhc is ngui. So let's move this to `validateArgsNgui`